src/main/js/dashboard-frontend/src/index.tsx

```diff
@@ -59,11 +61,11 @@ class App extends Component<any, AppState> {
     return (
       <HashRouter>
         <AppLayout
-          className="app"
```

Comment:
Why doesn't it need this class anymore?

Reply:
it was never used anyway, and they deprecated the className option too